QuickOSM/core/parser/preset_parser.py

```diff
@@ -0,0 +1,90 @@
+""" parser of the default presets file """
```

Comment:
Use capital string to make the same with other docstring. It's always using capital letter, full stop as well.